### PR TITLE
Enhance hyperlink security in project layout

### LINKED ISSUE #6048: Resolve CodeQL alert 14 "Potentially unsafe external link"
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to resolve the alert "Potentially unsafe external link" which appears in the CodeQL [alert 14](https://github.com/hackforla/website/security/code-scanning/14) by adding the attribute `rel="noopener noreferrer"`

### Action Items
- [x] The following item is required for GitHub to establish tracking between this issue and the alert.  No action is required.  You may simply check the checkbox.  If you do follow the link to learn more about CodeQL alerts, DO NOT DISMISS THE ALERT.  
- [x] https://github.com/hackforla/website/security/code-scanning/14
- [x] Open the file `_layouts/project.html` in your IDE
- [x] Replace
```
<a target="_blank" href='{{ item.url }}'> {{ item.name }}</a>
```
with
```
<a target="_blank" href='{{ item.url }}' rel="noopener noreferrer"> {{ item.name }}</a>
```
- [x] Using Docker, check the page remains the same in mobile, tablet, and desktop views as on the current website (See 2 in the Resources/Instructions section below)

### Merge Team
- [ ] When this issue is completed, check off the issue in #5129
  
### Resources/Instructions
1. [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
2. Webpage - sample project page: https://www.hackforla.org/projects/311-data
3. This issue is part of #5129

### PR DESCRIPTION
**Pull request by JackCasica**
Fixes #6048

### What changes did you make?
  - Updated the first hyperlink in the project's layout to include `rel="noopener noreferrer"` attributes,

### Why did you make the changes (we will use this info to test)?
  - Adding this attribute improves security and performance by preventing the newly opened page from controlling the page that linked to it. This change guards against potential exploits such as `window.opener` API misuse and reduces the chance of performance issues on the originating page.
  

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
- N/A

### CODE REVIEW

**Comment by mshirk2** [comment]
Review ETA: 2/5/24 6pm (PT) 
Availability: Weekdays 10-6pm (PT)

File: _layouts/project.html [hunk 1]
```diff
@@ -42,8 +42,7 @@ <h5 class='status-text'>{{ page.status }}</h5>
                     <strong>Links: </strong>
                     {% for item in page.links %}
                     {% if counter == 1 %}
-                    <a target="_blank" href='{{ item.url }}'> {{ item.name }}</a>
-                    {% assign counter = counter | minus:1 %}
+                    <a target="_blank" href='{{ item.url }}' rel="noopener noreferrer"> {{ item.name }}</a>                    {% assign counter = counter | minus:1 %}
```
Comment (mshirk2) [inline _layouts/project.html]: @JackCasica it appears an additional line was changed in your edits. For cleanliness, please return `{% assign counter = counter | minus:1 %}` to its own line.

**Review by mshirk2 — CHANGES_REQUESTED** [review]
@JackCasica please see inline comment for a small change needed. Once resolved, feel free to re-request PR review.

**Comment by njackman-2344** [comment]
Availability: 2/2 Friday 4pm-7pm, 2/3 Saturday 10am-1pm

ETA: 2/3 Saturday EOD

File: _layouts/project.html (hunk 1 above)
Comment (JackCasica) [reply _layouts/project.html]: Thanks @mshirk2  ~  It's resolved and ready for a re-review.

**Comment by njackman-2344** [comment]
@JackCasica 
For Comment consistency, please keep screenshot: Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes) and update us underneath :)

I pulled code and saw it on same line consistency in Docker. Thanks!

**Comment by JackCasica** [comment]
> @JackCasica For Comment consistency, please keep screenshot: Screenshots of Proposed Changes Of The Website (if any, please do not screen shot code changes) and update us underneath :)

Hello @njackman-2344 ~ I updated the comment to include the requested header and added `n/a` underneath. Is this the correct way to comment for no visual changes? Thanks.

**Review by njackman-2344 — APPROVED** [review]
Branches look good

issue linked

code looks good and seen  screen in different sizes in docker
 
Thanks!

**Review by mshirk2 — APPROVED** [review]
@JackCasica Thanks for making that change, and well done with this pull request!

- The new branch is merging into the correct branch 'gh-pages'
- Original issue is linked
- PR description is clear and correct
- The specified change has been correctly implemented

**Pull request merged**